ross/fluid_flow/tilting_pad.py

```diff
@@ -0,0 +1,1606 @@
+                np.pi * (self.c_r ** 2) * np.sqrt((self.Wx ** 2) + (self.Wy ** 2))
+            )
+
+        elif self.sommerfeld_type == 2:
+            S = 1 / (
+                2
+                * ((self.L / (2 * self.R)) ** 2)
+                * (np.sqrt((force_x ** 2) + (force_y ** 2)))
+            )
+
+        Ss = S * ((self.L / (2 * self.R)) ** 2)
+
+        return Ss
+
+
+if __name__ == "__main__":
+
+    phi = 30 * np.pi / 180
+    E = 0.5
+
+    # optim values from legacy codes
+    x = np.array(
+        [
+            0.000401905634685165,
+            0.000210288009621476,
+            0.000136772568561263,
+            0.000273217426488742,
+            0.000537108459033454,
+            0.000574699109242178,
+        ]
+    )
+    # psi_pad = x
+    psi_pad = np.array([x[0], x[1], x[2], x[3], x[4], x[5]])
+    npad = 6
+
+    # Radial clearance
+    Cr = 250e-6
+
+    # Oil tank temperature
+    Tcuba = 40
+
+    alpha = 0  # * (2 * np.pi * 5) * alpha
+
+    # Geometric parameters for the bearing --------------------------------------------
+    # Journal radius
+    R = 0.5 * 930e-3
+
+    # Pad radius
+    Rs = 0.5 * 934e-3  # [m]
+
+    # Pad thickness
+    esp = 67e-3  # [m]
+
+    # Pad arc
+    betha_s = 25  # [degree]
+
+    # Pivot position (arc pivot/arc pad)
+    rp_pad = 0.6
+
+    # Bength of bearing
+    L = 197e-3  # [m]
+
+    # Angular position of the pivots [degrees]
+    sigma = np.array([0, 60, 120, 180, 240, 300])
+
+    # Bearing loading
+    fR = 90.6e3  # [N]
+
+    # Rotor speed
+    wa = 300  # [rpm]
+
+    # Reference temperature
+    T_ref = Tcuba  # [Celsius]
+
+    # Thermal properties for the oil ----------------------------------------------------
+    # Thermal conductivity
+    kt = 0.07031 * np.exp(484.1 / (Tcuba + 273.15 + 474))  # [J/s.m.C]
+
+    # Specific heat
+    Cp = (16.5 * np.exp(-2442 / (Tcuba + 273.15 + 829.1))) * 1e3  # [J/kgC]
+
+    # Specific mass
+    rho = 0.04514 * np.exp(9103 / (Tcuba + 273.15 + 2766)) * 1e3  # [kg/m**2]
+
+    # Reference viscosity
+    # mi_ref=0.0752
+    mi_ref = 5.506e-09 * np.exp(5012 / (Tcuba + 273.15 + 0.1248))  # [N.s/m**2]
+
+    # Bearing Position ---------------------------------------------------------
+
+    #  Discretized Mesh ------------------------------------------------------
+
+    # Number of volumes in theta direction
+    ntheta = 48
+
+    # Number of volumes in x direction
+    nX = ntheta
+
+    # Number of volumes in z direction
+    nZ = 48
+
+    # Number of volumes in neta direction
+    nN = 30
+
+    mancal = Tilting(
+        R,
+        Rs,
+        esp,
+        betha_s,
+        rp_pad,
+        L,
+        sigma,
+        fR,
+        wa,
+        kt,
+        Cp,
+        rho,
+        mi_ref,
+        E,
+        phi,
+        psi_pad,
+        npad,
+        Cr,
+        alpha,
+        Tcuba,
+        ntheta,
+        nX,
+        nZ,
+        nN,
+    )
+    mancal.run()
+    # mancal.coefficients()
```

Comment:
Add a test file and move this part of the code to there.

Reply:
Alright, this is the next task in line. It should be done in a bit.

Reply:
Done! I will run a few tests with it and leave some default values for conference of code validity.